src/styles/components/_form.scss

```diff
@@ -192,11 +194,8 @@ select:not([multiple]) {
   .form-group & {
     @include di-variant("caret-down");
 
-    // sass-lint:disable no-vendor-prefixes
     -moz-appearance: none;
```

Comment:
waarom hebben we deze vendor-prefixes? Zit dit niet in de build?

Reply:
Er zit geen autoprefixer in de build. En deze styling zal wel gebaseerd zijn op Bootstrap.

Reply:
dat werpt dan wel een vervolgvraag op: zouden we geen autoprefixer willen/moeten hebben?

Reply:
Lijkt mij een prima idee